Neos.ContentRepository.Core/Classes/CommandHandler/PendingProjections.php

```diff
@@ -96,20 +97,23 @@ public static function fromProjectionsAndEventsAndSequenceNumber(
         SequenceNumber $highestCommittedSequenceNumber
     ): self {
         $sequenceNumberInteger = $highestCommittedSequenceNumber->value - $events->count() + 1;
-        $pendingProjections = Projections::create();
+        $pendingProjectionsArray = [];
         $sequenceNumberPerProjection = [];
         foreach ($events as $event) {
             foreach ($allProjections as $projection) {
+                if ($event instanceof DecoratedEvent) {
```

Comment:
This can be moved to the outer loop, can't it?

Reply:
True, good catch